### DISCUSSION TITLE
Praise! and a question about blank species stats.

### DISCUSSION THREAD

**Discussion by ade-secrutiny** [mcguirepr89/BirdNET-Pi#422] [Q&A] [answered]
Firstly, loving this - thanks so much for the effort to bring it to the Pi. Secondly - I have a blank screen when I select "Species Stats". Just the green background of the site. The installation only went live today, is it possible that not enough data is gathered yet or do I have an issue? Running on Rpi4B (4G RAM). Thank you!

**Comment by ade-secrutiny** [comment, marked as answer]
eurgh. Ignore me! problem was between the monitor and the chair... Well, to be precise, it was how Chrome renders the page. Works fine on Safari, FFox and Edge! thanks.

**Reply by CaiusX** [reply]
Hmm, was designed on Chrome monitor, but it's really difficult to get it looking decent on the variety of browsers, screen sizes and resolutions out there.

**Reply by ade-secrutiny** [reply]
Could well be me - or some of the script control / privacy stuff I'm running in Chrome - I'll have a detailed look later and post anything interesting, but for now I am very happy with my Safari based experience :-) Thank you!

**Reply by CaiusX** [reply]
May be a security issue, @ehpersonal38 has just pushed an update, maybe try update your installation- Tools, System menu.

**Reply by ade-secrutiny** [reply]
Thanks - will do, and I'll let you know

**Reply by ehpersonal38** [reply]
You could try this:
1) Open your BirdNET-Pi web interface on Chrome
2) Press CTRL + SHIFT + J to reveal the console
3) Take a screenshot of the whole page and paste it here
4) Press CTRL + SHIFT + I to reveal the network tab
3) Take a screenshot of the whole page and paste it here

After that we'll be able to see if there are any errors and hopefully determine what went wrong.

**Reply by ade-secrutiny** [reply]
Ok problem solved - it was my Ghostery privacy extension taking exception to some scripts. I've set the local pi site to be trusted and it's working fine in Chrome now. For interest and completeness, here's a screenshot of what Ghostery was blocking: 
![Screenshot 2022-07-04 at 19 11 10](https://user-images.githubusercontent.com/101126486/177202729-552694c1-daf7-455f-b309-dbee4596f734.png)

**Reply by ade-secrutiny** [reply]
Thanks for suggestions!

**Reply by ehpersonal38** [reply]
Ah, you caught a case of the overzealous chrome extension. Glad to see everything is working now. Enjoy your BirdNET-Pi!

**Reply by ade-secrutiny** [reply]
Thank you!
